hack/collector-default

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# -------------------------------------------------------------------------------- #
+# Description                                                                      #
+# -------------------------------------------------------------------------------- #
+# collctor-default will collect host information as package for support-bundle-kit #
+# e.g., hostname, syslog, dmesg etc                                                #
+# -------------------------------------------------------------------------------- #
+
+HOST_PATH=$1
+BUNDLE_DIR=$2
+
+cd ${BUNDLE_DIR}
+
+# -------------------------------------------------------------------------------- #
+# host information                                                                 #
+# -------------------------------------------------------------------------------- #
+mkdir -p hostinfos
+cd hostinfos
+
+echo "hostname : " `cat ${HOST_PATH}/etc/hostname` > hostinfo
+echo "system_info : " `uname -a` >> hostinfo
+
+# Boot Configuration
+cat ${HOST_PATH}/boot/config-$(uname -r) > boot_config
+
+# CPU, Memory and Process usage status
+top -b -n 1 > cpu_memory_process_status
```

Comment:
One-second top data seems not really useful. Do we really need it?

Reply:
Got it, I think CPU and memory should be based on the usage rate over a period of time, which would be more helpful. 

I think this can be provided by users when needed, I will remove this first.

Reply:
Yeah. If user can provide long-term data, it will be more helpful.